bool[] pos_control
int32[] servo_indices
---
int32 err
